versioned_docs/version-2.x/advanced/analytics/server-side-tagging-tracking.mdx

```diff
@@ -0,0 +1,186 @@
+---
+sidebar_position: 4
+title: Server-side tracking / tagging
+description:
+  This page will guide you trough the setup of Server-side tracking and tagging
+  services
```

Comment:
**issue:** I think having a dedicated page isn't relevant. Addingwell is a service like others, and as far as FC is concerned it still is client-side tracking.

**suggestion:** move the How-To under a dedicated subsection of the existing ["Common plugins"](https://deploy-preview-903--heuristic-almeida-1a1f35.netlify.app/docs/2.x/advanced/analytics/plugins#common-plugins) section.